app/rtc_compare.py

```diff
@@ -0,0 +1,126 @@
+'''
+Compare the two RTC products if they are equivalent
+Part of the codes are copied from PROTEUS SAS
+'''
+
+import argparse
+
+import h5py
+import numpy as np
+
+RTC_S1_PRODUCTS_ERROR_TOLERANCE = 1e-6
+
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='Compare two DSWx-HLS products',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    # Inputs
+    parser.add_argument('input_file',
+                        type=str,
+                        nargs=2,
+                        help='Input data in HDF5')
+
+    return parser
+
+def compare_hdf5(hdf_obj_1, hdf_obj_2, key_in='/', compare_attr=True):
```

Comment:
Not sure if you need this flag `compare_attr`. I think you always want to compare attributes, right?

Reply:
Makes sense, although the attributes are not the processing results but descriptions, units, etc. I'll take the flag out.